frontend/challedu-app/src/App.css

```diff
@@ -1,38 +1,39 @@
-.App {
-  text-align: center;
+
+#root{
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  background: url(../src/images/wallpaper.svg);
+  background-size: cover;
+  font-family:'Segoe UI';
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.welcome{
+  text-align: center;
```

Comment:
use bootstrap class `text-center`

Reply:
its the same thing mohammedAlabdali96, because they took vanila css and put it in a library doesnt make it better.

Reply:
I think though @mohammedAlabdali96 is right even though this is not compulsory. This has been open for too long so we'll make changes later